core/genesis_write.go

```diff
@@ -513,6 +513,10 @@ func GenesisToBlock(g *types.Genesis, tmpDir string) (*types.Block, *state.Intra
 		withdrawals = []*types.Withdrawal{}
 	}
 
+	if g.Config != nil && g.Config.Bor != nil && g.Config.Bor.IsAgra(g.Number) {
+		withdrawals = []*types.Withdrawal{}
```

Comment:
Hmm. I'm under impression that withdrawals are not included into the Agra fork. If that's the case, `withdrawals` should be left as `nil` and this branch is unnecessary.

Reply:
Yeah, withdrawals is a eth specific thing. We have a blank array in bor so cannot make an update here as it will result into hash mismatch

Reply:
Why have blank array in Bor instead of no withdrawals at all (=`nil` array)?